.lint

```diff
@@ -1,3 +1,7 @@
 exclusions:
  template-job-rule:
  panel-job-instance-rule:
+ template-datasource-rule:
+  reason: CI failing because label has different caps, this was changed in grafana-builder
+ panel-datasource-rule:
+  reason: The new Grafonnet promotes the use of datasources at the query level. This should probably end up in the linter as a valid option.
```

Comment:
The linter wants that the Panel has a datasource set that matches the template variable, this is debatable. Panels do not really deal with datasources, the value at the panel level is passed through by the UI to the query targets.

When you only use one data source, this doesn't matter that much but when combining queries from multiple data sources on the same panel, then you have to set it to `-- Mixed --` on the Panel level. 

This is the what the new Grafonnet does, from a jsonnet/code perspective you'll see that the data source is configured on the query rather than the panel.

Reply:
Attempting to upgrade and fix linting here: https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/891